.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,20 @@
+name: Continuous Integration
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
```

```suggestion
  push
```

You can get rid of the branch match - it's default to run against all branches if none are specified!

---

Removing the colon from push gives a syntax error